src/components/RepositoryCardContainer.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import { StoreState } from '../store';
+import { githubRepositoryDetail } from '../modules/githubDetailRepository/action';
+import './RepositoryDetailCard.css';
+import { RepositoryCard } from './RepositoryCard';
+
+export const RepositoryCardContainer: React.FC = () => {
+    const dispatch: any = useDispatch();
+    const {item} = useSelector((state: StoreState) => state.githubRepository);
+    const {repoParam} = useParams();
+    const [owner, repositoryName] = repoParam?.split('+') || [];
+
+    useEffect(() => {
+        if(owner && repositoryName && !item) {
+            dispatch(githubRepositoryDetail(owner, repositoryName));
+        }
+    }, [item, owner, repositoryName]);
+
+    return (
```

Comment:
```jsx
return item && <RepositoryCard item={item} />;
```

In general, it would be cool UX to display a RepositoryCard skeleton while the data is loading, instead of displaying a card only if item exists.

https://ant.design/components/skeleton/#components-skeleton-demo-active